src/models/tab-group.js

```diff
@@ -64,7 +64,7 @@ export default class TabGroup {
         updatedAt: new Date().toJSON(),
       })
         .then(() => {
-          chrome.runtime.sendMessage(constants.CHANGE);
+          browser.runtime.sendMessage(constants.CHANGE);
```

Comment:
This messaging API should probably be moved into a service to keep the border between extension and browser in a single place